bfcfg

```diff
@@ -37,19 +37,26 @@ bfcfg_rc=0
 cfg_file=/etc/bf.cfg
 log_file=/tmp/bfcfg.log
 dump_mode=0
+dump_all_mode=0
```

Comment:
A more extensive way is to reuse the dump_mode, so later we don't have to add more variables, such as

dumo_mode:
1: existing dump
2: all
3: selective dump to be defined later.

Reply:
I think we still need a new script parameter. Optional param arguments are not POSIX standard (for example being able to pass both `bfcfg -d` with no arguments and also `bfcfg -d 2`.

So two solutions could be:
1. `bfcfg -d <dump_mode>` : required `-d` argument
2. `bfcfg -d -l <dump_level>` : new param (replace `-a|--all` with `-l|--dump-level`)

Since the first option can break backward compatibility (e.g. someone uses `bfcfg -d` with no arguments in a script), I'm leaning towards the second option. Thoughts?

Reply:
Thanks!, yes #2 seems a lot better